plugins/action/zos_fetch.py

```diff
@@ -157,10 +157,16 @@ def run(self, tmp=None, task_vars=None):
         src = self._connection._shell.join_path(src)
         src = self._remote_expand_user(src)
 
+        if not (dest.startswith("/")):
```

Comment:
I know we know why this was added, I was there at the office working with you on this. Could you add some comments in this section so future us know why this is in place ?